### DISCUSSION TITLE
Not able to rename the `ui` package: Can't resolve 'foobar\package.json'

### DISCUSSION THREAD

**Discussion by xairoo** [vercel/turborepo#1107] [Help] [answered]
I just started to use turbo basic example and I tried to rename the `ui` package. Without that change all worked.

What I have done:
- deleted `apps/docs`
- renamed folder `ui` to `foobar`
- updated `packages/foobar/package.json` to `"name": "foobar",`
- updated `apps/web/next.config.js` -> `ui` to `foobar`
- updated `apps/web/package.json` -> `ui` to `foobar`
- And of course `import { Button } from "foobar";`

Results in:
```
Error: next-transpile-modules - an unexpected error happened when trying to resolve "foobar". Are you sure the name of the module you are trying to transpile is correct, and it has a package.json with a "main" or an "exports" field?
web:dev: Error: Can't resolve 'foobar\package.json' in 'S:\Projects\my-turborepo\apps\web'
```

`next.config.js`
```
const withTM = require("next-transpile-modules")(["foobar"]);

module.exports = withTM({
  reactStrictMode: true,
});

```

There isn't much I can do wrong. What have I missed?

**Comment by xairoo** [comment, marked as answer]
I have to run `npm i` after doing this ;)

**Reply by hamudeshahin** [reply]
Yep, `npm i` worked for me too

**Comment by hamudeshahin** [comment]
I have changed that but while i am trying to run `yarn install` There is an error occurs which is: 
```
error Couldn't find package "geeks-ui@*" required by "docs@0.1.0" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

I have already changed the name of package to the new name in `apps/docs/package.js`
What exactly  is the problem here??
